### ISSUE TITLE
Shipping fees per combination

### ISSUE THREAD

**Issue by vincentbz** [PrestaShop/PrestaShop#11305]
Salut les poilus !

**Is your feature request related to a problem? Please describe.** 
In a product page, there is the possibility to add a specific shipping fee (shipping tab). But when I add combinations to my products, all of them have the same shipping fees, there is no possibility to change the shipping fee per combination.

**Describe the solution you'd like**
As the field "shipping fees" in the Shipping Tab of a product page, it would be great to have the same in each combination. So that I can determinate my small tee shirt costs 3€ to ship, 4€ for the medium, 5€ shipping fees for the large.

**Specs**
add an "Impact on shipping fees" field in the combinations.
In the shipping tab, the "shipping fees" field already has a tooltip and its content is essential. We need to keep it.

![image](https://github.com/PrestaShop/PrestaShop/assets/113457393/8fb9780a-a987-493d-b095-30dd9d6cece3)

**Additional context**

![image](https://user-images.githubusercontent.com/7567060/48136478-0ddc8280-e2a0-11e8-9308-52dd062f9ca1.png)

**Comment by khouloudbelguith** [comment]
Hi @vincentbz,

Thanks for your suggestion.
@eternoendless, @marionf, @PrestaShop/prestashop-core-developers what do you think?
Thanks!

**Comment by vincentbz** [comment]
![deadpool](https://user-images.githubusercontent.com/7567060/48138824-695d3f00-e2a5-11e8-900b-48468cdf70c3.gif)

**Comment by PierreRambaud** [comment]
Totally legit!

**Comment by BilalDr** [comment]
Hi everyone, 
it's a big subject, 
So you wanted to add shipping fees for each combinations. 

Currently without combinations we can define shipping fees 
on the panel shipping > shipping fees section. 
If we have combinations, the shipping fee defined is setted for every combinations. 
 
For your idea, 
Firstly,
On the page shipping, add an alert next to Shipping fees input 
![shippingfees-alerte](https://user-images.githubusercontent.com/40759259/48712169-63ece680-ec0d-11e8-9cde-0f30c860e20f.png)
The alert should be like this : 
with this wording :  
>This field as an impact on all combinations, if you want to have specific shipping fees, change it on combination edit panel

![alert](https://user-images.githubusercontent.com/40759259/48712238-8aab1d00-ec0d-11e8-8066-3d10ebe89336.PNG)

 And, if we want to define shipping fees per combinations, 
we should add on product with combinations on product page > combinations panel > edit > price and impact 
a field with shipping fees. 
![shippingfees-input](https://user-images.githubusercontent.com/40759259/48712302-b7f7cb00-ec0d-11e8-8972-7c4dfc894a3b.png)


After this, you will be able to define shipping fees for every combination you have. 

What do you think about the alert and the wording ? 
@NadiaCamard 
@LouiseBonnard

**Comment by LouiseBonnard** [comment]
@BilalDr, how about: _Need specific shipping fees? Go to the combination tab to precise/edit/change it!_

**Comment by rblaurin** [comment]
What about keeping the page shipping unmodified and add a field "Impact on shipping fees ($)" in combinations?

**Comment by marionf** [comment]
Yes, as said by BilalDr we can add the field in the edit combination page and add an information message in Shipping tab below the shipping fees input

**Comment by ElodieOS** [comment]
Hello @MatShir @TristanLDD 

What's about this issue? Is it still relevant?

**Comment by MatShir** [comment]
The need is still relevant

**Comment by ElodieOS** [comment]
Hello everyone!
I am agree with @rblaurin, we can keep the shipping page unchanged and just add an "Impact on shipping fees" field in the combinations. 
In the shipping tab, the "shipping fees" field already has a tooltip and its content is essential. We need to keep it. Adding an extra help text can overload the interface and is not essential, in my opinion. 

Wdyt? 

![image](https://github.com/PrestaShop/PrestaShop/assets/113457393/8fb9780a-a987-493d-b095-30dd9d6cece3)

**Comment by jrstarenlard** [comment]
Hello,

What is the status of this issue? Is there any plan to include this field in the combinations?
